can/bus.py

```diff
@@ -223,14 +238,19 @@ def wrapped_stop_method(remove_task=True):
                     pass
             original_stop_method()
 
-        task.stop = wrapped_stop_method
+        setattr(task, "stop", wrapped_stop_method)
```

Comment:
Interesting... was this to satisfy mypy?

Reply:
Yeah, I was getting the following:

```
can/bus.py:241: error: Cannot assign to a method
```

It appears that mypy doesn't [really](https://github.com/python/mypy/issues/2427) have good [support](https://github.com/python/mypy/issues/708) for this sort of patching, so the workaround is to use `setattr` or `type: ignore`.